backend/grant/proposal/models.py

```diff
@@ -272,19 +274,52 @@ def __init__(
         self.stage = stage
 
     @staticmethod
-    def validate(proposal):
+    def simple_validate(proposal):
         title = proposal.get('title')
         stage = proposal.get('stage')
         category = proposal.get('category')
+
         if title and len(title) > 60:
             raise ValidationException("Proposal title cannot be longer than 60 characters")
         if stage and not ProposalStage.includes(stage):
             raise ValidationException("Proposal stage {} is not a valid stage".format(stage))
         if category and not Category.includes(category):
             raise ValidationException("Category {} not a valid category".format(category))
 
+    def validate_publishable_milestones(self):
+        payout_total = 0.0
+        for i, milestone in enumerate(self.milestones):
+
+            if milestone.immediate_payout and i != 0:
+                raise ValidationException("Only the first milestone can have an immediate payout")
+
+            if len(milestone.title) > 60:
+                raise ValidationException("Milestone title must be no more than 60 chars")
+
+            if len(milestone.content) > 6000:
+                raise ValidationException("Milestone title must be no more than 6000 chars")
```

Comment:
Milestone content, not title. And we limit it to 200 on the frontend (create/utils.ts:127)

Reply:
Was actually going for content here, to set some upper bound.

Reply:
I'll do 50000

Reply:
Unless you meant that we limit content to 200 chars on the frontend? That doesn't seem like a lot for the content. Maybe you meant brief?

Reply:
These are milestones, not proposals.

Reply:
Definiely down to up it, but they're supposed to be pretty small. Just something like

> **Mainnet launch**
> This is when it goes live. We should have a fully audited codebase, with desktop, mobile, and web apps ready to use on mainnet.

Reply:
Right, what am I thinking.